dokku-unlock.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh -l
+
+script_dir=$(dirname $0)
+
+. "$script_dir/setup-ssh.sh"
+
+echo ""
+echo "Unlocking Dokku deployment";
+
+app=$(echo $DOKKU_REPO | sed -e 's/.*\///')
+ssh -p ${ssh_port:=22} dokku@$ssh_host apps:unlock $app
```

Comment:
`apps:unlock` [requires dokku >= 0.11.6](http://dokku.viewdocs.io/dokku/deployment/application-management/#unlocking-app-deploys), you should maybe write that somewhere in the readme?

Reply:
Also, this assumes that the dokku user on remote in actually called `dokku`.
I don't know if it could ever be changed.

Probably not, since the official `dokku_client.sh` assumes that the user is `dokku` as well: https://github.com/dokku/dokku/blob/master/contrib/dokku_client.sh#L149.
So it's probably fine.

Reply:
> `apps:unlock` [requires dokku >= 0.11.6](http://dokku.viewdocs.io/dokku/deployment/application-management/#unlocking-app-deploys), you should maybe write that somewhere in the readme?

Good idea !